pkg/api/login_test.go

```diff
@@ -740,5 +740,5 @@ func mockAuthenticateUserFunc(user *models.User, authmodule string, err error) {
 	}
 }
 func resetAuthenticateUserFunc() {
-	login.AuthenticateUserFunc = login.AuthenticateUser
+	// login.AuthenticateUserFunc = login.AuthenticateUser
```

Comment:
Fix this before merge: in or out, let's not leave commented out code, at least not without a comment about it 🙂